### DISCUSSION TITLE
Connection Refused

### DISCUSSION THREAD

**Discussion by dbrianhendrix** [mcguirepr89/BirdNET-Pi#541] [Q&A]
This is new. I woke up this morning and started seeing these errors and EVERY recording -- still investigating:

07:53:45 birdnet_analysis.sh[3416745]: analyze.py --i /home/birder/BirdSongs/September-2022/11-Sunday/2022-09-11-birdnet-12:42:15.wav --o /home/birder/BirdSongs/September-2022/11-Sunday/2022-09-11-birdnet-12:42:15.wav.csv --lat 35.9870 --lon -87.1451 --week 34 --overlap 0.0 --sensitivity 1.25 --min_conf 0.7 --birdweather_id "IN_USE"
07:53:45 birdnet_analysis.sh[3437694]: Traceback (most recent call last):
07:53:45 birdnet_analysis.sh[3437694]:   File "/usr/local/bin/analyze.py", line 13, in <module>
07:53:45 birdnet_analysis.sh[3437694]:     client.connect(ADDR)
07:53:45 birdnet_analysis.sh[3437694]: ConnectionRefusedError: [Errno 111] Connection refused
07:53:45 birdnet_analysis.sh[3416745]: RECORDING_LENGTH set to 15
07:53:45 birdnet_analysis.sh[3416745]: analyze.py --i /home/birder/BirdSongs/September-2022/11-Sunday/2022-09-11-birdnet-12:42:30.wav --o /home/birder/BirdSongs/September-2022/11-Sunday/2022-09-11-birdnet-12:42:30.wav.csv --lat 35.9870 --lon -87.1451 --week 34 --overlap 0.0 --sensitivity 1.25 --min_conf 0.7 --birdweather_id "IN_USE"
07:53:45 birdnet_analysis.sh[3437700]: Traceback (most recent call last):
07:53:45 birdnet_analysis.sh[3437700]:   File "/usr/local/bin/analyze.py", line 13, in <module>
07:53:45 birdnet_analysis.sh[3437700]:     client.connect(ADDR)
07:53:45 birdnet_analysis.sh[3437700]: ConnectionRefusedError: [Errno 111] Connection refused
07:53:45 birdnet_analysis.sh[3416745]: RECORDING_LENGTH set to 15
07:53:46 birdnet_analysis.sh[3416745]: analyze.py --i /home/birder/BirdSongs/September-2022/11-Sunday/2022-09-11-birdnet-12:42:45.wav --o /home/birder/BirdSongs/September-2022/11-Sunday/2022-09-11-birdnet-12:42:45.wav.csv --lat 35.9870 --lon -87.1451 --week 34 --overlap 0.0 --sensitivity 1.25 --min_conf 0.7 --birdweather_id "IN_USE"
07:53:46 birdnet_analysis.sh[3437706]: Traceback (most recent call last):
07:53:46 birdnet_analysis.sh[3437706]:   File "/usr/local/bin/analyze.py", line 13, in <module>
07:53:46 birdnet_analysis.sh[3437706]:     client.connect(ADDR)
07:53:46 birdnet_analysis.sh[3437706]: ConnectionRefusedError: [Errno 111] Connection refused
07:53:46 birdnet_analysis.sh[3416745]: RECORDING_LENGTH set to 15
07:53:46 birdnet_analysis.sh[3416745]: analyze.py --i /home/birder/BirdSongs/September-2022/11-Sunday/

**Comment by mcguirepr89** [comment]
Hi, @dbrianhendrix -- could you please open an issue for this issue? Thanks!
